client/src/components/__tests__/CheckBoxWithLabel.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import CheckBoxWithLabel from '../CheckBoxWithLabel';
+
+describe('<CheckBoxWithLabel />', () => {
+  const requiredProps = { name: 'remember-me', label: 'Remember Me' };
+  it('should return an <input> of type checkbox', () => {
+    const wrapper = mount(<CheckBoxWithLabel {...requiredProps} />);
+
+    expect(wrapper.exists('input')).toBe(true);
+    expect(wrapper.find('input').props().type).toBe('checkbox');
+  });
+
+  it('checkbox should have passed props', () => {
+    const props = { a: 123, b: 345, c: 678 };
+    const wrapper = mount(<CheckBoxWithLabel {...requiredProps} {...props} />);
+    expect(wrapper.find('Checkbox').props()).toMatchObject(props);
+  });
+
+  test('should have passed label', () => {
```

Comment:
@niinpatel Please try and be consistent with the test function. Prefer `it`, but don't mix them like this.

Reply:
This one slipped past my attention, thanks for pointing out. I generally prefer `test` because vscode autocompletes for me. 

I don't think it matters though, both have same meaning semantically and functionally.

Reply:
Fixed this.